Mathlib/LinearAlgebra/Matrix/Adjugate.lean

```diff
@@ -417,6 +417,210 @@ theorem adjugate_fin_two (A : Matrix (Fin 2) (Fin 2) α) :
 theorem adjugate_fin_two_of (a b c d : α) : adjugate !![a, b; c, d] = !![d, -b; -c, a] :=
   adjugate_fin_two _
 #align matrix.adjugate_fin_two_of Matrix.adjugate_fin_two_of
+--adjugate !![a, b, c; d, e, f; g, h, i]
+theorem adjugate_fin_three (A : Matrix (Fin 3) (Fin 3) α) :
+    adjugate A =
+    !![(A 1 1) * (A 2 2) - (A 1 2) * (A 2 1),
+     -((A 0 1) * (A 2 2)) + (A 0 2) * (A 2 1),
+      (A 0 1) * (A 1 2) - (A 0 2) * (A 1 1);
+     -((A 1 0) * (A 2 2)) + (A 1 2) * (A 2 0),
+      (A 0 0) * (A 2 2) - (A 0 2) * (A 2 0),
+       -((A 0 0) * (A 1 2)) + (A 0 2) * (A 1 0);
+      (A 1 0) * (A 2 1) - (A 1 1) * (A 2 0),
+       -((A 0 0) * (A 2 1)) + (A 0 1) * (A 2 0),
+        (A 0 0) * (A 1 1) - (A 0 1) * (A 1 0)] := by
```

Comment:
Most of the `()`s are not needed here (they're not in the `fin_two` version either); I also suggest tidying the indentation a little to highlight where each row begins:
```suggestion
    !![
      A 1 1 * A 2 2 - A 1 2 * A 2 1,
        -(A 0 1 * A 2 2) + A 0 2 * A 2 1,
        A 0 1 * A 1 2 - A 0 2 * A 1 1;
      -(A 1 0 * A 2 2) + A 1 2 * A 2 0,
        A 0 0 * A 2 2 - A 0 2 * A 2 0,
        -(A 0 0 * A 1 2) + A 0 2 * A 1 0;
      A 1 0 * A 2 1 - A 1 1 * A 2 0,
        -(A 0 0 * A 2 1) + A 0 1 * A 2 0,
        A 0 0 * A 1 1 - A 0 1 * A 1 0] := by
```